### PR TITLE
chore: remove deprecated_className and slotClassNames 

### PR DESCRIPTION
**Pull request by mnajdova**
This PR removed the static `className` (`deprecated_className`) and `slotClassNames` props from the fluent components, in order to eliminate the component dependencies inside the theme definition.

**BREAKING CHANGES**

Replace all component's  `className` (`deprecated_className`) and `slotClassNames` props with the dedicated consts: `componentNameClassName` and `componentNameSlotClassNames`.

```jsx
// before
import { Button, MenuItem } from '@fluentui/react-northstar';

console.log(Button.className);
console.log(MenuItem.slotClassNames.indicator);

// after
import { buttonClassName, menuItemSlotClassNames } from '@fluentui/react-northstar';

console.log(buttonClassName);
console.log(menuItemSlotClassNames.indicator);
```

### CODE REVIEW

File: packages/fluentui/react-northstar/src/components/Accordion/Accordion.tsx [hunk 1]
```diff
@@ -102,8 +102,6 @@ class Accordion extends AutoControlledComponent<WithAsProp<AccordionProps>, Acco
 
   static deprecated_className = accordionClassName;
```
Comment (pompomon) [inline packages/fluentui/react-northstar/src/components/Accordion/Accordion.tsx:103]: You said in the description "Replace all component's className (deprecated_className)", but I see them still in place, is that OK?

Comment (pompomon) [reply packages/fluentui/react-northstar/src/components/Accordion/Accordion.tsx:103]: Or are we keeping `deprecated_className` for class components?

Comment (mnajdova) [reply packages/fluentui/react-northstar/src/components/Accordion/Accordion.tsx:103]: We still need them internally for the components that extending the `UIComponent` and `AucontrolledComponents`, but we don't want users to rely on those.

File: packages/fluentui/react-northstar/src/components/Flex/FlexItem.tsx [hunk 2]
```diff
@@ -58,7 +58,7 @@ const applyStyles = (
   // if element is DOM element
   if (typeof element.type === 'string') {
     return React.cloneElement(element, {
-      className: cx(element.props.deprecated_className, classes.root),
+      className: cx(element.props.className, classes.root),
```
Comment (pompomon) [inline packages/fluentui/react-northstar/src/components/Flex/FlexItem.tsx:61]: Please how would that work with class components, which still have `deprecated_className` ?

Comment (mnajdova) [reply packages/fluentui/react-northstar/src/components/Flex/FlexItem.tsx:61]: This was unintended change where we are using the prop className, not the static field, but I accidentally replaced it with `deprecated_className` in one of the prev PRs.

**Pull request merged**